common/api/appui-react.api.md

```diff
@@ -5265,6 +5297,8 @@ export interface Widget {
     readonly id: string;
     // (undocumented)
     readonly label?: string | ConditionalStringValue;
+    // @alpha
+    readonly layouts?: WidgetLayouts;
```

Comment:
Looking at writing the Content documentation and I wonder if we shouldnt name this differently, we already have Content Layouts (and StandardContentLayouts in appui-abstract) Adding another `layout` name in the mix will add to the already confusing naming we have. 
Sadly, I dont have a great idea to replace this name..
`uiOrganisation`
`uiConfig`
`destinations`
`where`
`home`
`slot`
`at`
`in`
`showIn`

Reply:
I don't really like the proposed alternative names or could come up with a better alternative. My vote is to keep the `layouts` property name as is, since the `ContentLayout` type seems unique enough.

Reply:
Fine, we can always revisit if needed and feedback reveals any confusion.